docs/core/diagnostics/observability-applicationinsights.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Example: Use OpenTelemetry with Azure Monitor and Application Insights"
+description: An walkthrough of how to use OpenTelemetry in .NET to export telemetry to Application Insights
+ms.date: 8/21/2024
+ms.topic: conceptual
+---
+
+# Example: Use OpenTelemetry with Azure Monitor and Application Insights
+
+There are many commercial Application Performance Management (APM) systems available to choose from. In Azure, the primary application-monitoring product is [Application Insights](/azure/azure-monitor/app/app-insights-overview?tabs=net), which is a part of Azure Monitor. One of the advantages of an integrated APM product is that it can correlate the different observability data sources. Application Insights has rich views and analysis capabilities.
+
+## 1. Adding the Application Insights distro
+
+To make the ASP.NET experience with Azure Monitor easier, a wrapper package (called a Distro in OTel parlance) is provided that does most of the heavy lifting of configuring OpenTelemetry.
+
+This example is based off the [OTLP walkthrough](./observability-OTLP-example.md). Follow the steps 1-5 to create the application code using the OTLP exporter. In this example, we will extend the code to send data to Application Insights.
+
+Take the same project from [Step 5](./observability-OTLP-example.md#5-configure-opentelemetry-with-the-correct-providers) and add the following NuGet package:
+
+``` xml
+<ItemGroup>
+  <PackageReference Include="Azure.Monitor.OpenTelemetry.AspNetCore" Version="1.3.0-beta.1" />
+</ItemGroup>
+```
+
+> [!Note]
```

Comment:
```suggestion
> [!NOTE]
```

Reply:
Elsewhere its using the mixed case, should it be all caps everywhere?

Reply:
Yes, it should be all caps everywhere.